### ISSUE TITLE
Suggestion: Looks to complicated...

### ISSUE THREAD

**Issue by Mqxx** [jodeks-datapacks/Too-Expensive-Removed#1]
Hey, nice concept. Only problem is that it looks a little bit complicated.

You could try to target the items in the anvil directly instead of letting the player shuffle items in their inventory around to the specific slot...

**Comment by Jodekq** [comment]
Yup I got the same idea but I think it's not possible to target the items in the anvil directly, as it's not block entity. Or am I missing something?

**Comment by Mqxx** [comment]
Not sure, we need to check.
